src/hooks/index.ts

```diff
@@ -5,6 +5,7 @@ import { useDisableTableSortersIfLoading } from "hooks/useDisableTableSortersIfL
 import { useOnClickOutside } from "hooks/useOnClickOutside";
 import { useFilterInputChangeHandler } from "hooks/useFilterInputChangeHandler";
 import { useStatusesFilter } from "hooks/useStatusesFilter";
+
```

Comment:
Automatically adds a new line between imports and components @tgrander

Reply:
heheheheeheh!!!!! 😍